plugins/modules/packaging/os/apt_repo.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Mikhail Gordeev
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'core'}
+
+DOCUMENTATION = '''
+---
+module: apt_repo
+short_description: Manage APT repositories via apt-repo
+description:
+  - Manages APT repositories using apt-repo tool.
+  - See U(https://www.altlinux.org/Apt-repo) for details about apt-repo
+notes:
+    - This module works on ALT based distros.
+    - Does NOT support checkmode, due to a limitation in apt-repo tool.
+version_added: "2.8"
```

Comment:
FWIW `migrate.py` removed all `version_added` from `DOCUMENTATION` during the migration so this should be removed too.

Reply:
Oh, that's interesting

Reply:
done

Reply:
It didn't remove them for return values, though..

Reply:
Correct, `EXAMPLES` and `RETURN` were left intact. Unfortunately there was only so much we were able to automate in time.